guide/daily-spending-wallet/sending.md

```diff
@@ -198,7 +198,7 @@ On the Lightning network, payments are passed between nodes to get from the send
 
 **Lightning service provider fees**
```

Comment:
Could we bump the size of these subheadings? Not just here but also the section above which has recipient, amount and metadata. 

A ### or #### would be good

Reply:
```suggestion
**Lightning service fees**
```

So this covers LSPs and LWSs